templates/telco/osm/onboarding-network-functions.html

```diff
@@ -0,0 +1,187 @@
+      {{ image (
+        url="https://assets.ubuntu.com/v1/e8f2394f-Onboarding+2.svg",
+        alt="",
+        width="1000",
+        height="180",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="col-12">
+      <h2>Lifecycle management</h2>
+      <p>The goal of onboarding is to create a package which can fulfil the stages to manage the lifecycle of network services. The package should  include network function instantiation, scaling in/out, healing or recovery, updating, operating and termination.</p>
+      <p>Using private and public cloud infrastructure at the same time allows you to optimise your CapEx and OpEx costs.</p>
+    </div>
+		<hr style="margin-bottom: 2rem;" />
+		<div class="col-12 u-hide--small u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/68971b5f-Onboarding+3.svg",
+        alt="",
+        width="1000",
+        height="500",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+			<h2>VNF onboarding in OSM Hackfests</h2>
+      <p>Canonical regularly participates in the events organized by the OSM community and ETSI’s Centre for Testing and Interoperability. For events like Hackfests and NFV Plugtests Canonical also offers a partner cloud where multiple VNF vendors can deploy and test the interoperability of their Telco network functions. It is also available if VNF vendors want to test their network functions individually before migrating to production environments.</p>
+    </div>
+    <div class="col-12">
+			Network Functions onboarded graph
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru-accent is-dark">
+  <div class="row">
+    
+    <div class="col-12">
+      <div class="p-heading-icon--small">
+        <h2>Tell us about your network function</h2>
+        <p>Canonical can help you automate your network function onboarding to Charmed OSM. Please choose the relevant options for your network functions to see how it can work for you.</p>
+        <p><a href="https://charmed-osm.com/#get-in-touch" class="p-button--positive">Start now</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+		<h2>Let us help you in onboarding</h2>
+    <div class="col-6 p-card">
+      <h4><a class="p-link--external" href="https://charmed-osm.com/">Charmed OSM deployment support</a></h4>
+			<hr />
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Supported with <a href="/ua">Ubuntu Advantage</a></li>
+        <li class="p-list__item is-ticked">Availability of managed and professional services</li>
+        <li class="p-list__item is-ticked">ESM - Security updates for Ubuntu LTS for additional 3-5 years</li>
+      </ul>
+    </div>
```

Comment:
Could you use the correct card classes please: https://vanillaframework.io/docs/patterns/card#header

Reply:
I added the `card-title` I believe that's what you meant? `p-list` seems to be fine